odbc_version.i

```diff
@@ -1,5 +1,5 @@
 #define MAJOR_VERSION 11
-#define MINOR_VERSION 0
+#define MINOR_VERSION 1
 #define PATCH_VERSION 0
 #define BUILD_SERIAL_NUMBER 0001
 #define VERSION_STRING "11.0.0.0001"

```

Comment:
isn't it 11.1.0.0001?

Reply:
I missed it by mistake.
updated VERSION_STRING.